### PR TITLE
Addition of Service Mesh Pattern

### LINKED ISSUE #5: [Markdown] Add Service Mesh Patterns to project list
#### Current Behavior
The https://github.com/service-mesh-patterns project is missing from the project list.

Screen Shot 2021-09-21 at 8 46 24 AM

#### Desired Behavior
Include this project alongside the others.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by Shivanshu97i**
Signed-off-by: Shivanshu Singh <singhshivanshu97@gmail.com>

**Description**
added service mesh pattern

This PR fixes #5

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -56,6 +57,20 @@ alt="Service Mesh Landscape Ecosystem" align="left"/></a>
...
+</p>
+<br />
+<br />
+<h2><a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a></h2>
+<a href="https://meshery.io"><img src="https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/.github/readme/images/service-mesh-pattern.png"
+style="margin:10px;" width="125px" 
+alt="Meshery - Service Mesh Patterns" align="left" /></a>
+
+<a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a> help you get the most out of Linkerd, Consul, App Mesh, Istio, Kuma, Open Service Mesh or <a href="https://layer5.io/service-mesh-landscape">any service mesh</a>, including those using Envoy and use of WebAssembly. Each pattern can be used as a template and is customizable.
+<br /><br /><br />
```
Comment (leecalcote) [inline profile/README.md]: Were you able to get the "help wanted" shield in here, too?

**Comment by leecalcote** [comment]
Thanks for this, @Shivanshu97i

File: profile/README.md (hunk 1 above)
Comment (Shivanshu97i) [reply profile/README.md]: No, I did not add help wanted shield wlll add asap. Can you please provide the hyperlink to be associated with it

Comment (debo19) [reply profile/README.md]: @Shivanshu97i https://img.shields.io/github/issues-search?label=help%20wanted&query=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+repo%3Aservice-mesh-patterns%2Fservice-mesh-patterns this should work.

Comment (Shivanshu97i) [reply profile/README.md]: Thanks, Will add this immediately

Comment (debo19) [reply profile/README.md]: @Shivanshu97i do attach a screenshot of the final preview.

Comment (Shivanshu97i) [reply profile/README.md]: Issue 5
@debo19 Sorry, The images are not loading don't know why but the final preview looks something like this

File: profile/README.md [hunk 2]
```diff
@@ -59,6 +60,20 @@ The <a href="https://layer5.io/landscape">Service Mesh Landscape</a> is a compre
 <a href="https://github.com/search?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+repo%3Alayer5io%2Flayer5" alt="Help wanted in Service Mesh Landscape">
 <img alt="Help wanted in Service Mesh Landscape" src='https://img.shields.io/github/issues-search?label=help%20wanted&query=is%3Aissue+is%3Aopen+label%3A"help+wanted"+repo%3Alayer5io%2Flayer5'></a> 
 <br />
+</p>
+<br />
+<h2><a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a></h2>
+<a href="https://meshery.io"><img src="https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/.github/readme/images/service-mesh-pattern.png"
```
Comment (debo19) [inline profile/README.md]:
```suggestion
<a href="https://github.com/service-mesh-patterns/service-mesh-patterns"><img src="https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/.github/readme/images/service-mesh-pattern.png"
```

**Review by debo19 — COMMENTED** [review]
Looks good, just this minor change then we're good to go here.

**Comment by Shivanshu97i** [comment]
@debo19 Even though I signed my coomits why is the DCO bot rejecting my commits.

**Review by debo19 — APPROVED** [review]
LGTM 🎉 Thanks @Shivanshu97i

**Pull request merged**